src/css/controls/flags/audioplayer.less

```diff
@@ -34,6 +34,7 @@
         .jw-button-container {
             padding-right: 3px;
             padding-left: 0;
+            justify-content: flex-start;
```

Comment:
lgtm. Might want to try replacing with margin-right: auto and see how that handles the edge cases. It will change the layout slightly (re: everything after it will be pushed all the way to the right edge of the containing element), but might prevent some of the edge cases we discussed earlier.

Reply:
@zetagame So I spot checked with live and dvr streams and their layout seems fine (Not sure if there were other edge cases to check?). 
 The other thing I noticed with `flex-start` was that with very small players other control buttons might start getting cut off, but the play button would remain visible so it was still slightly usable. 
`margin-right:auto` did work when added to `.jw-text-alt` during ad playback. but sometimes it would still be a little jumpy if the player got too small.